### ISSUE TITLE
possible mistake in VPC/STAC creation

### ISSUE THREAD

**Issue by wiesehahn** [r-lidar/lasR#24]
Hi, I am just starting to play around with lasR. 

I recognised that in the file that `write_vpc()` creates there is an attribute "datatime", but I think in STAC it should be "datetime" (see e.g. https://github.com/stac-extensions/pointcloud/blob/main/examples/example-autzen.json#L86).

**Comment by wiesehahn** [comment]
Two other things I just recognised, 
- "pc:count" is 0 for all features (files) in my VPC, is there any possible reason this might have been popluated like this?
- "pc:encoding" is "?" for all features. It seems to have been removed from the specification (https://github.com/stac-extensions/pointcloud/issues/8).

**Comment by Jean-Romain** [comment]
Support of vpc is partial and requires a review. Thank you for your contribution. I already know that I did not implement the full VPC specification. The specifications are complex and spread into multiple sub-specifications... For example, I think it is mandatory to export the bounding box in long/lat and it is not mandatory to export the bbox in the CRS of the point cloud. It should be the opposite, in my opinion...

>  there is an attribute "datatime"

That looks like a typo I'll check.

> "pc:count" is 0 for all features (files) in my VPC

It should not. `write_vpc()`  is tested in the unit tests and produces the correct values. Maybe a bug with a LAS format (LAS 1.4 ?). Please report with a reproducible example.

> "pc:encoding" is "?" 
 
I created a VPC file with PDAL and then copied the stucture of the file. PDAL probably produces this one. I will verify what is the purpose of this attribute.

**Comment by Jean-Romain** [comment]
This file included with the package has been created with pdal wrench https://github.com/r-lidar/lasR/blob/main/inst/extdata/bcts/bcts.vpc. It has `pc:encoding = "?"`

**Comment by Jean-Romain** [comment]
`datatime` was a typo. Fixed.

**Comment by Jean-Romain** [comment]
Reproduced `pc:count = 0` with

```r
library(lasR)
f <- system.file("extdata", "las14_prf6.laz", package="rlas")
pipeline <- write_vpc(tempfile(fileext = ".vpc"))
exec(pipeline, on = f)
```

**Comment by Jean-Romain** [comment]
Fixed `pc:count` for LAS 1.4
